drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -132,41 +133,48 @@ static const uint8_t dbm_to_rx_sens[] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
 
 uint16_t at86rf2xx_get_addr_short(const at86rf2xx_t *dev)
 {
-    return (dev->netdev.short_addr[0] << 8) | dev->netdev.short_addr[1];
+    network_uint16_t addr;
+
+    addr.u8[1] = at86rf2xx_reg_read(dev, AT86RF2XX_REG__SHORT_ADDR_0);
+    addr.u8[0] = at86rf2xx_reg_read(dev, AT86RF2XX_REG__SHORT_ADDR_1);
+
+    return addr.u16;
 }
 
 void at86rf2xx_set_addr_short(at86rf2xx_t *dev, uint16_t addr)
 {
-    dev->netdev.short_addr[0] = (uint8_t)(addr);
-    dev->netdev.short_addr[1] = (uint8_t)(addr >> 8);
+    /* addr is in network byte order */
+    network_uint16_t ap;
```

Comment:
What does `ap` stand for? in the original code [here](https://github.com/RIOT-OS/RIOT/pull/10401/files#diff-a40616a9bbe7880b61a333a3d5c554acL156) I could imagine it meaning "address pointer", but this is not a pointer anymore... :-/